api/jobs.yaml

```diff
@@ -201,85 +201,12 @@ definitions:
       labels:
         type: object
         description: Custom job labels with string values
-      logs:
-        type: object
-        description: Map of type of log file to its location
-      tasks:
-        type: array
-        items:
-          $ref: '#/definitions/TaskMetadata'
       failures:
         type: array
         items:
           $ref: '#/definitions/FailureMessage'
-  JobStatus:
-    type: string
-    enum:
-      - Submitted
-      - Running
-      - Aborting
-      - Failed
-      - Succeeded
-      - Aborted
-  TaskMetadata:
-    description: Task level metadata
-    properties:
-      inputs:
-        type: object
-        description: Mapping of input fully qualified names to stringified values
-      # TODO: Define a generalized enum for this. For details, see
-      # https://github.com/DataBiosphere/job-manager/pull/14.
-      executionStatus:
-        type: string
-        description: Backend-specific task status.
-      start:
-        type: string
-        format: date-time
-        description: Start datetime of the task execution in ISO8601 format with milliseconds
-      end:
-        type: string
-        format: date-time
-        description: End datetime of the task execution in ISO8601 format with milliseconds
-      name:
-        type: string
-        description: The name of the job
-      executionId:
-        type: string
-        description: Backend-specific job ID
-      failures:
-        type: array
-        items:
-          $ref: '#/definitions/FailureMessage'
-      returnCode:
-        type: integer
-        description: Task execution return code
-      stdout:
-        type: string
-        description: Path to the standard output file for this task
-      stderr:
-        type: string
-        description: Path to the standard error file for this task
-      attempts:
-        type: integer
-        description: Number of times the task was run
-      jobId:
-        type: string
-        description: >
-          Id of the job corresponding to this task, if this task is a nested
-          child job. Expresses a child:parent relationship between this job
-          and the job containing this task.
-  FailureMessage:
-    description: Failure messages
-    required:
-      - failure
-    properties:
-      failure:
-        type: string
-        description: The failure message
-      timestamp:
-        type: string
-        format: date-time
-        description: The time at which this failure occurred
+      extendedFields:
```

Comment:
opt: I'm on the fence, but looking at this again, "fields" feels slightly redundant. We could consider making this `extended` instead or `extensions` for more readable references:

`job.extendedFields.userId`
`job.extended.userId`
`job.extensions.userId`

Both here and for query (though the model name could remain as-is). WDYT?